packages/opencv/lib/index.js

```diff
@@ -335,7 +282,10 @@ async function getImagesMatches(img1Data, img2Data, options = {}) {
         width: rect2.width,
         height: rect2.height,
       });
-      result.visualization = await jimpImgFromCvMat(visualization).getBufferAsync(Jimp.MIME_PNG);
+      result.visualization = base64Encode(
```

Comment:
Is it a breaking change? I remember we had a call of `convertVisualizationToBase64` and `visualization` itself was always stored as raw buffer

Reply:
it is if you consider this to be a public API, but the only place this code is called is in this module.  the code which calls this function encoded the result; I just moved the encoding to in here since it seemed appropriate.

Reply:
yep, my point is we need to be quite careful with such breaking changes to not do too much damage to the depending code. I would rather keep it as is then have to do with all the hassle about finding and fixing all the dependencies that may potentially be broken. Especially when we are talking about more stylistic change like this one rather than some mandatory bugfix or update